src/Facades/ChildProcess.php

```diff
@@ -18,6 +19,8 @@ class ChildProcess extends Facade
 {
     protected static function getFacadeAccessor()
     {
-        return \Native\Laravel\ChildProcess::class;
+        self::clearResolvedInstance(Implement::class);
```

Comment:
Right now, when using the facade multiple times during the same request, chances are high we get a exception about overriding readonly properties.

``` php
ChildProcess::artisan('main:loop', 'alias');
ChildProcess::get('alias'); // -> This errors
```

This is a side effect because the base Facade keeps a static internal dictionary of previously resolved instances & returns one from there if available. This effectively makes ChildProcess a singleton (no matter if the binding is defined otherwise in the container)

The line I added here circumvents this intentional facade behaviour. The impact of instantiating a ChildProcess object again is very very minimal. 

There are other ways to go about this, for example by binding the Facade to a Factory that creates a fresh ChildProcess instance. The current workaround, while unconventional, might suffice for now.